### PR TITLE
bootstrapped react, test and lint

### LINKED ISSUE #1: Bootstrap React component and repository
**What needs to be done? Why does it need to be done?**
Add the basic infrastructure needed to create a React component in version control. The deliverable artifact here is simply a React component that can be dropped into a separate application. However, it's important to be able to test this ourselves, so we need some minimal "app" that we can drop the component into and run _locally_ for testing. 

**Additional context**
We don't need any infrastructure for hosting or deploys. We just need to be able to run/test the component (and sub-components that get created).

### LINKED ISSUE #2: Bootstrap testing and linting
**What needs to be done? Why does it need to be done?**
For this component, we need just basic unit testing and linting. This can follow the standards/conventions from pulse-dashboard. We should be able to run these via `yarn test` and `yarn lint`, respectively. Unlike pulse-dashboard, there should not be a lint whitelist: _every_ file should pass lint in this new repository.

### PR DESCRIPTION
**Pull request by Wmaileq**
## Description of the change

- bootstrapped CRA
- added linters
- added test util settings and sample test
- added first component

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

> Closes #1, Closes #2 

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
+{
+  "name": "supervision-success-component",
```
Comment (jessex) [inline package.json:2]: A few more bits of metadata to add in here:
```
{
  ...
  "description": "A tool for modeling the impact of an increase in supervision success on the broader justice system",
  "repository": "git@github.com:Recidiviz/supervision-success-component.git",
  "author": "Recidiviz <team@recidiviz.org>",
  "license": "GNU General Public License v3",
  ...
}
```

**Review by jessex — CHANGES_REQUESTED** [review]
Code looks good, thanks @Wmaileq! Heads up, though, `yarn test` and `yarn lint` both fail for me with a completely fresh install. Do you see the same failures?
```
$ yarn test
 FAIL  src/SupervisionSuccess/__tests__/SupervisionSuccess.test.js
  ● Test suite failed to run

    Cannot find module '../providers/SupervisionSuccessModelProvider' from 'SupervisionSuccess.test.js'

       5 | import SupervisionSuccessModelProvider from "../providers/SupervisionSuccessModelProvider";
       6 | 
    >  7 | jest.mock("../providers/SupervisionSuccessModelProvider", () => ({
         |      ^
       8 |   SupervisionSuccessModelProvider: jest.fn().mockReturnValue(null),
       9 | }));
      10 | 

      at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:259:17)
      at Object.<anonymous> (src/SupervisionSuccess/__tests__/SupervisionSuccess.test.js:7:6)

Test Suites: 1 failed, 1 total
Tests:       0 total
Snapshots:   0 total
Time:        1.429s
Ran all test suites.

Watch Usage: Press w to show more.

$ yarn lint
yarn run v1.22.4
$ eslint ./src

/Users/clementinejacoby/Documents/projects/supervision-success-component/src/SupervisionSuccess/__tests__/SupervisionSuccess.test.js
  5:45  error  Unable to resolve path to module '../providers/SupervisionSuccessModelProvider'  import/no-unresolved
  5:45  error  Missing file extension for "../providers/SupervisionSuccessModelProvider"        import/extensions

✖ 2 problems (2 errors, 0 warnings)

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by Wmaileq** [comment]
> Code looks good, thanks @Wmaileq! Heads up, though, `yarn test` and `yarn lint` both fail for me with a completely fresh install. Do you see the same failures?
> 
> ```
> $ yarn test
>  FAIL  src/SupervisionSuccess/__tests__/SupervisionSuccess.test.js
>   ● Test suite failed to run
> 
>     Cannot find module '../providers/SupervisionSuccessModelProvider' from 'SupervisionSuccess.test.js'
> 
>        5 | import SupervisionSuccessModelProvider from "../providers/SupervisionSuccessModelProvider";
>        6 | 
>     >  7 | jest.mock("../providers/SupervisionSuccessModelProvider", () => ({
>          |      ^
>        8 |   SupervisionSuccessModelProvider: jest.fn().mockReturnValue(null),
>        9 | }));
>       10 | 
> 
>       at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:259:17)
>       at Object.<anonymous> (src/SupervisionSuccess/__tests__/SupervisionSuccess.test.js:7:6)
> 
> Test Suites: 1 failed, 1 total
> Tests:       0 total
> Snapshots:   0 total
> Time:        1.429s
> Ran all test suites.
> 
> Watch Usage: Press w to show more.
> 
> $ yarn lint
> yarn run v1.22.4
> $ eslint ./src
> 
> /Users/clementinejacoby/Documents/projects/supervision-success-component/src/SupervisionSuccess/__tests__/SupervisionSuccess.test.js
>   5:45  error  Unable to resolve path to module '../providers/SupervisionSuccessModelProvider'  import/no-unresolved
>   5:45  error  Missing file extension for "../providers/SupervisionSuccessModelProvider"        import/extensions
> 
> ✖ 2 problems (2 errors, 0 warnings)
> 
> error Command failed with exit code 1.
> info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
> ```

fixed

**Review by jessex — APPROVED** [review]
Good to go! 🚀

**Pull request merged**